german-council-of-economic-experts.csl

```diff
@@ -0,0 +1,310 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" class="in-text" default-locale="de-DE" demote-non-dropping-particle="sort-only">
```

Comment:
With `default-locale="de-DE"`, the style is fixed to German. If you also would like to use the style in English, you can delete this.